### PR TITLE
cypress setup in piro

### PR DESCRIPTION
**Pull request by rajimtri**
TRI-31004-Setting up cypress project for piro

### CODE REVIEW

**Review by Pradeepvarma333 — APPROVED** [review]

File: e2etests/cypress/support/e2e.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import './commands'
```
Comment (shilpasivanesan-tri) [inline e2etests/cypress/support/e2e.js:1]: This file can be removed , as  we will be adding ssl cert

File: e2etests/package-lock.json
Comment (shilpasivanesan-tri) [inline e2etests/package-lock.json:1]: can be removed

File: e2etests/report.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+report.generate( {
+    jsonDir: "./cypress/json-logs/",
+    reportPath : "./cucumber-report/",
+    metadata: {
+        browser: {
+            name :"chrome",
+            version: 60,
+        },
+        device: "local test machine",
```
Comment (shilpasivanesan-tri) [inline e2etests/report.js:13]: will it be local device ? and also the platform is fixed only to windows

**Review by shilpasivanesan-tri — CHANGES_REQUESTED** [review]

File: e2etests/cypress/support/e2e.js (hunk 1 above)
Comment (rajimtri) [reply e2etests/cypress/support/e2e.js:1]: This is a placeholder as part of the setup. Can remove this if its not required as part of entire implementation of test scripts for PIRO

File: e2etests/report.js (hunk 2 above)
Comment (rajimtri) [reply e2etests/report.js:13]: This will be addressed with github workflow

**Review by shilpasivanesan-tri — APPROVED** [review]

**Pull request merged**